app/(auth)/_layout.tsx

```diff
@@ -5,6 +5,12 @@ export default function GetStart() {
   return (
     <Stack>
       <Stack.Screen name="Login" options={{ headerShown: false }} />
+      <Stack.Screen name="KnucklesPage" options={{ title: 'What is Knuckles' }} />
+      <Stack.Screen name="WildFirePage" options={{ title: 'How WildFire Happen' }} />
+      <Stack.Screen name="SafetyPrecautionPage" options={{ title: 'Safety Precaution' }} />
+      <Stack.Screen name="AfterFirePage" options={{ title: 'What To Do After A Fire Happen' }} />
+      <Stack.Screen name="AppTutorialPage" options={{ title: 'App Tutorial' }} />
+      
```

Comment:
These should be in separate file in separate folder. Because this is a separate route